src/Worlds.scss

```diff
@@ -95,6 +95,113 @@ body.contrast-theme {
     }
 }
 
+/* Haunted House */
```

Comment:
`Worlds.scss` is a little out of alphabetical order (Landmarks is in the wrong place). For this PR, could you please move the new styles for Haunted House down to between Deep Ocean and Jungle please? That way just Landmarks is in the wrong place and can be moved later.

Reply:
I moved the styles for both Haunted House and Landmarks so the file is now alphabetical by world name.